spec/models/company_spec.rb

```diff
@@ -9,15 +9,26 @@
     it { is_expected.to validate_presence_of :email }
   end
 
-  describe '.all' do
-    let(:company_a) { double(:company_a, name: 'xyz') }
-    let(:company_b) { double(:company_b, name: 'abc') }
-    let(:company_c) { double(:company_c, name: 'MNO') }
+  describe '#all' do
```

Comment:
I agree with @Drenmi let's use `Company` object here instead of a double (otherwise we'd have to also test `<=>` method since it's stubbed here)

Reply:
done! :)